src/components/Grid/Grid.e2e.ts

```diff
@@ -0,0 +1,25 @@
+import * as eyes from 'eyes.it';
+import { browser } from 'protractor';
+import {
+  createStoryUrl,
+  waitForVisibilityOf,
+} from 'wix-ui-test-utils/protractor';
+import { gridTestkitFactory } from '../../testkit/protractor';
+
+describe('grid', () => {
+  const storyUrl = createStoryUrl({
+    kind: 'Components',
```

Comment:
We want this to be under a new category of `Bookings` components.
So I guess you can change the `kind` to `Bookings/Components` or just `Bookings`.
We're not really familiar with the entire spec of this component, and we're not sure this should be part of the library. However, we don't want to obstruct your development, and we prefer having the component here, so others might want to use it, but under a different category.

Reply:
putting under Bookings